tensorboard/webapp/app_routing/route_config_types.ts

```diff
@@ -47,9 +47,39 @@ export interface ConcreteRouteDef {
 }
 
 export interface RedirectionRouteDef {
-  routeKind: null;
+  routeKind?: null;
```

Comment:
Do note that this is to be dropped in internal TensorBoard when no redirection specifies routeKind.